package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "ember-frost-icons",
+  "version": "0.0.1",
+  "description": "An inline-svg component and svg library for Frost projects",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint .",
+    "start": "ember server",
+    "build": "ember build",
+    "test": "npm run-script lint && ember test"
+  },
+  "repository": "git@github.com:ciena-frost/ember-fr-svg.git",
+  "engines": {
+    "node": ">= 0.10.0"
+  },
+  "author": "Steven Glanzer (https://github.com/sglanzer)",
+  "contributors": [
+    "Roxanne Panthaky (https://github.com/rox163)",
+    "Olufemi Adeojo",
+    "Gosia Hyndman (https://github.com/vesper2000)",
+    "Adam Meadows (https://github.com/job13er)"
+  ],
+  "license": "MIT",
+  "devDependencies": {
+    "broccoli-asset-rev": "^2.2.0",
+    "ember-cli": "1.13.13",
+    "ember-cli-app-version": "^1.0.0",
+    "ember-cli-blanket": "0.8.0",
+    "ember-cli-content-security-policy": "0.4.0",
+    "ember-cli-dependency-checker": "^1.1.0",
+    "ember-cli-github-pages": "0.0.6",
+    "ember-cli-htmlbars-inline-precompile": "^0.3.1",
+    "ember-cli-inject-live-reload": "1.3.1",
+    "ember-cli-mocha": "0.9.7",
+    "ember-cli-release": "0.2.8",
+    "ember-cli-sri": "^1.2.0",
+    "ember-cli-uglify": "^1.2.0",
+    "ember-disable-proxy-controllers": "^1.0.1",
+    "ember-export-application-global": "^1.0.4",
+    "ember-lodash": "^0.0.6",
+    "ember-truth-helpers": "1.1.0",
+    "ember-try": "~0.0.8",
+    "eslint": "^1.10.3",
+    "eslint-config-frost-standard": "^0.2.4",
+    "markdown-code-highlighting": "0.0.6"
```

Comment:
Another unrelated comment - I have a code parser / highlighter built using codemirror, we should look at getting this in place for the code samples